src/WholeBodyStateDisplay.cpp

```diff
@@ -151,12 +175,16 @@ WholeBodyStateDisplay::WholeBodyStateDisplay()
                                                 grf_category_, SLOT(updateGRFArrowGeometry()), this);
   grf_head_radius_property_ = new FloatProperty("Head Radius", 0.04, "Radius of the arrow's head, in meters.",
                                                 grf_category_, SLOT(updateGRFArrowGeometry()), this);
+  grf_locate_at_cop_property_ = new BoolProperty("Locate At Center of Pressure", true,
+                                                 "Collocate the ground reaction force with the foot's center of pressure.",
```

Comment:
Do not explain it as "foot's center of pressure". I believe it is better to be broad, i.e., "contact's center of pressure".

Reply:
Agreed.